sway-book/src/documentation/language/control-flow/match/single-line.md

```diff
@@ -0,0 +1,13 @@
+# Single Line Arm
+
+The following example demonstrates how a type can be matched on and its output is assigned to a variable. The assignment to a variable is optional.
+
+```sway
```

Comment:
The following example uses "catch_all" over "_". I believe the convention is to use the underscore and if you'd like a comment to mark that it is a catch all that would be better.